### ISSUE TITLE
[BUG] spark-excel library not working as a workspace library

### ISSUE THREAD

**Issue by LeonardoVallin** [crealytics/spark-excel#755]
### Is there an existing issue for this?

- [x] I have searched the existing issues

### Current Behavior

I have installed as a workspace library spark-excel_2.13-3.3.1_0.18.7 into databricks runtine 13.2 (includes Apache Spark 3.4.0, Scala 2.12) I'm reading a csv into a df, and when trying to create an excel file based on the df I'm getting the error [DATA_SOURCE_NOT_FOUND] Failed to find data source: com.crealytics.spark.excel. also tried with library version spark-excel_2.12-0.13.8 and get the same error, so I'm not sure if maybe the library is not compatible with the databricks cluster configuration, I was able to use it in a cluster configuration 10.4 LTS (includes Apache Spark 3.2.1, Scala 2.12)

### Expected Behavior

being able to create an excel file based on a csv dataframe. 

### Steps To Reproduce

df = spark.read.csv('abfss://container@datalakename.dfs.core.widnows.net/folderpath/filename.csv', header = true, sep = ',')
df.write.format("com.crealytics.spark.excel")\
	.option("header",'true')\
	.option("dataAddress", "'BlackDiamondActInclReport'!A1:X10000") \
        .mode("overwrite")\
        .save(save_folder_path)

### Environment

```markdown
- Spark version:3.4.0
- Spark-Excel version: 2.13-3.3.1_0.18.7
- OS: windows 11
- Cluster environment: databricks runtine 13.2 (includes Apache Spark 3.4.0, Scala 2.12)
```

### Anything else?

this is an error the last part of the error that seems that maybe we have missing some configuration. 

Py4JError: An error occurred while calling o601.getCause. Trace:
py4j.security.Py4JSecurityException: Method public java.lang.Throwable java.lang.ClassNotFoundException.getCause() is not whitelisted on class class org.apache.spark.SparkClassNotFoundException
	at py4j.security.WhitelistingPy4JSecurityManager.checkCall(WhitelistingPy4JSecurityManager.java:473)
	at py4j.Gateway.invoke(Gateway.java:305)
	at py4j.commands.AbstractCommand.invokeMethod(AbstractCommand.java:132)
	at py4j.commands.CallCommand.execute(CallCommand.java:79)
	at py4j.ClientServerConnection.waitForCommands(ClientServerConnection.java:195)
	at py4j.ClientServerConnection.run(ClientServerConnection.java:115)
	at java.lang.Thread.run(Thread.java:750)

**Comment by christianknoepfle** [comment]
Since spark's low level file I/O API changes a lot with minor versions, you always need the matching minor version used to build spark excel. The support for 3.4 is in the making, just waiting to get the pull request into the main code. So stay tuned

Best Christian

**Comment by nightscape** [comment]
Please try the newly released version 0.19.0 which contains the PR from @christianknoepfle that introduces Spark 3.4 compatibility.
